string task_uuid
string[] results